### DISCUSSION TITLE
No 'response_model_' routing option available to pass 'serialize_as_any=True' to pydantic

### DISCUSSION THREAD

**Discussion by ostefano** [fastapi/fastapi#11520] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@self.get(
    "/method",
    response_model_serialize_as_any=True,
)
async def get_method(
```

### Description

When specifying response model options there is no way to pass the `serialize_as_any` option to pydantic.

Note: it is possible to pass other pydantic options (e.g., `by_alias=True` by using `response_model_by_alias=True`)
I guess this is because this options has been added in pydantic 2.7.1 which has been released just a few weeks ago.

### Operating System

macOS

### FastAPI Version

0.110.0

### Pydantic Version

2.7.1

### Python Version

Python 3.12.3

**Comment by MrPranav101** [comment]
came here to say the same :) Please support this :)

**Comment by lean-zone** [comment]
FastAPI developers, please support this  as soon as possible.

**Comment by florianabel** [comment]
Any news here already?

**Comment by YuriiMotov** [comment]
I believe this shouldn't be added to FastAPI.
If you want to bypass the response validation, just return `JSONResponse` directly from the endpoint.

```python
from fastapi.responses import JSONResponse
from pydantic import BaseModel
from fastapi import FastAPI

app = FastAPI()

class User(BaseModel):
    name: str

class UserLogin(User):
    password: str

@app.get("/", response_model=User)
async def get_user():
    user = UserLogin(name="John Doe", password="secret123")
    return JSONResponse(content=user.model_dump())
```
